cmd/alertmanager/main.go

```diff
@@ -466,8 +466,17 @@ func run() int {
 			pipelinePeer = peer
 		}
 
+		active := make([]*notify.Receiver, 0, len(receivers))
```

Comment:
probably rename to `activeReceivers`?

Reply:
There's a map with the same name a few lines above, I'll try to find a better naming though.